pkgs/build-support/setup-hooks/strip.sh

```diff
@@ -61,13 +61,17 @@ stripDirs() {
 
     if [ -n "${paths}" ]; then
         echo "stripping (with command $cmd and flags $stripFlags) in $paths"
+        local striperr
+        striperr="$(mktemp 'striperr.XXXXXX')"
         # Do not strip lib/debug. This is a directory used by setup-hooks/separate-debug-info.sh.
         find $paths -type f -a '!' -path "$prefix/lib/debug/*" -print0 |
-            xargs -r -0 -n1 -P "$NIX_BUILD_CORES" $cmd $stripFlags 2>/dev/null || exit_code=$?
+            xargs -r -0 -n1 -P "$NIX_BUILD_CORES" $cmd $stripFlags 2>"$striperr" || exit_code=$?
         # xargs exits with status code 123 if some but not all of the
         # processes fail. We don't care if some of the files couldn't
         # be stripped, so ignore specifically this code.
-        [[ "$exit_code" = 123 || -z "$exit_code" ]]
+        [[ "$exit_code" = 123 || -z "$exit_code" ]] || (cat "$striperr" && exit 1)
```

Comment:
You probaby want unconditional fail: `(cat "$striperr"; exit 1)`. Otherwise if `cat` succeeds (and it usually does you won't get to `exit 1`). At least it does not fail for me in `extra-utils`.

Reply:
Ah, no. Please ignore. It never fails for me. The initial condition is fine.

Reply:
> this is easiest to do through a file
> 
> mktemp needs six X because busybox only accepts exactly six X

Please capitalize and punctuate commit messages.